dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -76,28 +76,28 @@ useAuthenticationDataForUnprotectedResource=Use authenticated identity for an un
 useAuthenticationDataForUnprotectedResource.desc=Specifies whether authentication data can be used when accessing an unprotected resource. The unprotected resource can access validated authenticated data that it previously could not access. This option enables the unprotected resource to call the getRemoteUser, isUserInRole, and getUserPrincipal methods to retrieve an authenticated identity. The equivalent custom property in the full application server profile is com.ibm.wsspi.security.web.webAuthReq=persisting.
 
 loginFormURL=Login form URL 
-loginFormURL.desc=Specifies the global URL of a form login page including the root context. The form login page must be part of the WAR file. If a form login application does not specify the form login page in the web.xml file, it uses the global form login URL.
+loginFormURL.desc=Specifies the global URL of a form login page including the root context. The form login page must be part of the WAR file. If a form login application does not specify the form login page in the web.xml file, it uses the global form login URL. This is required when overrideHttpAuthMethod attribute is set to FORM.
 
 loginErrorURL=Login error URL 
-loginErrorURL.desc=Specifies the global URL of a form login error page that includes the root context. The form login error page must be part of a WAR file. An application uses the global login error URL if the application uses form login authentication and does not specify either the form login page or the login error page in the auth-method element of the web.xml file.
+loginErrorURL.desc=Specifies the global URL of a form login error page that includes the root context. The form login error page must be part of a WAR file. An application uses the global login error URL if the application uses form login authentication and does not specify either the form login page or the login error page in the auth-method element of the web.xml file. This is required when overrideHttpAuthMethod attribute is set to FORM.
 
-allowAuthenticationFailOverToAuthMethod=Allow certificate authentication fail over
-allowAuthenticationFailOverToAuthMethod.desc= Specifies the authentication fail over method that will be used when the certificate authentication fails. Valid values are BASIC and FORM.
+allowAuthenticationFailOverToAuthMethod=Fail over method when certificate login fails
```

Comment:
When used as an adjective, Failover is one word. Change label to:
Failover method when certificate login fails

Reply:
Fixed.